src/app/subject/subject-list/subject-list.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="card mb-1" *ngFor="let subject of subjects">
+  <div class="card-block row">
+    <div class="card-body row align-items-center">
+      <div class="col">
+        <h6 class="card-subtitle text-muted">Subject</h6>
+        <a class="card-text" [routerLink]="['subjects', subject.id]">{{subject.name}}</a>
```

Comment:
This link is not working. Since you are already at `/subjects`, you only needo to link to subject.id